src/jit/gtlist.h

```diff
@@ -161,7 +161,7 @@ GTNODE(ADD_LO     , "+Lo"          ,1,GTK_BINOP)
 GTNODE(ADD_HI     , "+Hi"          ,1,GTK_BINOP)
 GTNODE(SUB_LO     , "-Lo"          ,0,GTK_BINOP)
 GTNODE(SUB_HI     , "-Hi"          ,0,GTK_BINOP)
-GTNODE(MUL_HI     , "*Hi"          ,1,GTK_BINOP)
+GTNODE(MUL_HI     , "*Hi"          ,1,GTK_BINOP) // A mul that returns the 2N bit result of an NxN multiply
```

Comment:
I think it would be good to add a comment in gtlist, since I had to do quite a bit of searching around to figure out what `GT_MULHI` is for - it would seem that this would be the right place ot describe how these are all related & different.

---
In reply to: [77717578](https://github.com/dotnet/coreclr/pull/7065#discussion_r77717578) [](ancestors = 77717578)